src/js/components/ExpandableGroup/ExpandableGroup.tsx

```diff
@@ -21,24 +23,37 @@ const propTypes = {
   groupHelperText: PropTypes.string || PropTypes.node,
   hasError: PropTypes.bool,
   onExpandedChange: PropTypes.func,
-  testId: PropTypes.string
+  testId: PropTypes.string,
+  deleteDisabled: PropTypes.bool,
+  deleteTooltip: PropTypes.string,
+  onToggle: PropTypes.func,
+  toggleValue: PropTypes.bool,
+  toggleTooltip: PropTypes.string,
```

Comment:
Would it make the attribute easier to read if it were named something like `toggleText` or `toggleTooltipText` ?

Reply:
Yeah, I should probably add `text` portion